netlify/functions/transDim.js

```diff
@@ -0,0 +1,167 @@
+require("dotenv").config();
+const nodemailer = require("nodemailer");
+const transDimConfig = require("../transDimConfig");
+
+// Function wrapper for sending email that returns a promise
+function sendEmail(mailOptions) {
+   console.log(mailOptions);
```

Comment:
If you dig through [netlify](https://app.netlify.com/sites/fervent-colden-5f0cd2/functions/transDim) you can find a console and when running the localhost netlify stack it will output to stdout.

This issue was about fixing a small configuration in the mailer that is in danger of becoming "re-write how email is sent from site" which I feel should be another ticket. The JS does *not* look good but it works so I consider this as acceptable within the context of the issue. 

Refactoring this code should be done on its own block of work and looked at within the context of a possible move of platform (I heard cloudflare being mentioned in chat).